nodeup/pkg/model/sysctls.go

```diff
@@ -151,6 +151,11 @@ func (b *SysctlBuilder) Build(c *fi.ModelBuilderContext) error {
 	}
 
 	if b.Cluster.Spec.IsIPv6Only() {
+		if b.Distribution == distributions.DistributionDebian11 {
+			sysctls = append(sysctls,
+				"# Enable Router Advertisements to get the default IPv6 route",
+				"net.ipv6.conf.ens5.accept_ra=2")
```

Comment:
Just for the record, the interface name depends the instance type. On e.g t2, this will be ens3. On upcoming instance types it can again be something else.